config.tf

```diff
@@ -1,39 +1,38 @@
 locals {
-  app_data_mode    = "${var.postgresql_address != "" ? "external_services" : "demo"}"
-  app_network_type = "${var.airgap_package_url != "" ? "airgap" : "online" }"
+  app_data_mode    = var.postgresql_address != "" ? "external_services" : "demo"
+  app_network_type = var.airgap_package_url != "" ? "airgap" : "online"
   install_type     = "${local.app_data_mode}-${local.app_network_type}"
 }
 
 # Settings for automated PTFE installation
 data "template_file" "repl_ptfe_config" {
-  template = "${local.rptfeconf[local.install_type]}"
-
-  vars {
-    hostname               = "${module.lb.endpoint}"
-    enc_password           = "${local.encryption_password}"
-    iact_subnet_list       = "${var.iact_subnet_list}"
-    iact_subnet_time_limit = "${var.iact_subnet_time_limit}"
-    pg_user                = "${var.postgresql_user}"
-    pg_password            = "${var.postgresql_password}"
-    pg_netloc              = "${var.postgresql_address}"
-    pg_dbname              = "${var.postgresql_database}"
-    pg_extra_params        = "${var.postgresql_extra_params}"
-
-    aws_access_key_id     = "${var.aws_access_key_id}"
-    aws_secret_access_key = "${var.aws_secret_access_key}"
-    s3_bucket_name        = "${var.s3_bucket}"
-    s3_bucket_region      = "${var.s3_region}"
+  template = local.rptfeconf[local.install_type]
```

Comment:
Yay, since we're on 0.12 we can move some of this to templatefile functions (later)!

Reply:
Yessss, I'm not a fan of how the various install type templates are broken out in this module vs others.